tests/robottelo/test_decorators.py

```diff
@@ -36,20 +38,49 @@ def test_not_smoke(self):
 class BzBugIsOpenTestCase(TestCase):
```

Comment:
Few pylint disables are intentionally added to make pylint happy keeping the test cases readable.